src/main/java/com/coreos/jetcd/internal/impl/ClientUtil.java

```diff
@@ -0,0 +1,132 @@
+package com.coreos.jetcd.internal.impl;
```

Comment:
it is probably the easiest to create a new pr and add the changes on top of the master branch; probably faster than resolve the merge conflict of this pr.